services/src/main/java/org/jfrog/artifactory/client/aql/AqlInclude.java

```diff
@@ -0,0 +1,29 @@
+package org.jfrog.artifactory.client.aql;
+
+import org.apache.commons.lang3.ArrayUtils;
+
+import java.util.Arrays;
+import java.util.StringJoiner;
+
+public class AqlInclude {
```

Comment:
Some of code lines in this methods seem not be aligned (not indented).

Reply:
fixed